src/Bridges/HttpDI/HttpExtension.php

```diff
@@ -94,8 +94,8 @@ public function afterCompile(Nette\PhpGenerator\ClassType $class)
 		}
 		if (strpos($csp, "'nonce'")) {
 			$csp = Nette\DI\ContainerBuilder::literal(
-				'str_replace(?, ? . $this->getService(?)->getNonce(), ?)',
-				["'nonce", "'nonce-", $this->prefix('response'), $csp]
+				'str_replace(?, ? . Nette\Utils\Random::generate(20), ?)',
```

Comment:
That is not base64; it should be base64_encode(Random::generate(???, "\x00-\xFF")) or Random::generate(???, "base64chars")